pkg/cli/config.go

```diff
@@ -183,6 +184,9 @@ func (c *Config) RegisterCommandLineFlags() {
 	if c.Flags.isSet(FlagVIN) {
 		flag.StringVar(&c.VIN, "vin", "", "Vehicle Identification Number. Defaults to $TESLA_VIN.")
 	}
+	if c.Flags.isSet(FlagBLE) {
+		flag.StringVar(&c.BtAdapterID, "btAdapter", "", "ID of the Bluetooth adapter to use (Linux only). Defaults to hci0.")
```

Comment:
It would be nice if Linux only flags only appeared on Linux systems, instead of having the Linux only disclaimer.